pytensor/tensor/ssignal.py

```diff
@@ -0,0 +1,38 @@
+import scipy.signal as scipy_signal
```

Comment:
Minor note that won't matter until the end of this PR but we probably want to do this import lazy to not slow down pytensor import. See https://github.com/pymc-devs/pytensor/pull/1268